components/constellation/constellation.rs

```diff
@@ -1825,15 +1827,12 @@ where
                 // the active media session.
                 // Events coming from inactive media sessions are discarded.
                 if self.active_media_session.is_some() {
-                    match event {
-                        MediaSessionEvent::PlaybackStateChange(ref state) => {
-                            match state {
-                                MediaSessionPlaybackState::Playing |
-                                MediaSessionPlaybackState::Paused => (),
-                                _ => return,
-                            };
-                        },
-                        _ => (),
+                    if let MediaSessionEvent::PlaybackStateChange(ref state) = event {
+                        match state {
+                            MediaSessionPlaybackState::Playing |
+                            MediaSessionPlaybackState::Paused => (),
+                            _ => return,
+                        };
```

Comment:
I wonder if this could just be:

```rust
if !matches!(state,  MediaSessionPlaybackState::Playing | MediaSessionPlaybackState::Paused) {
    return;
}
```

Reply:
Yes, I think that makes sense and is consistent with other changes. Thanks!